lib/base-controller.js

```diff
@@ -3,10 +3,12 @@
 var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
+var mixins = require('./middleware/mixins');
 
 var BaseController = function BaseController(options) {
   this.confirmStep = options.confirmStep || '/confirm';
   Controller.apply(this, arguments);
+  this.use(mixins);
```

Comment:
Do we need to use a middleware to do this? Afaik, we need only compile the templates once. I'm just concerned we are asking too much of app. Every time we add a middleware to the stack we slow down the request/response cycle